packages/vsce/src/SshConfigUtils.ts

```diff
@@ -649,23 +645,30 @@ export class SshConfigUtils {
                     await attemptConnection(config);
                     return { password: config.password };
                 } catch (error) {
-                    passwordAttempts++;
                     if (`${error}`.includes("FOTS1668")) {
                         vscode.window.showErrorMessage("Password Expired on Target System");
                         return undefined;
                     }
-                    vscode.window.showErrorMessage(`Password Authentication Failed (${passwordAttempts}/3)`);
+                    vscode.window.showErrorMessage(`Password Authentication Failed (${attempts + 1}/3)`);
                 }
             }
             return undefined;
         };
 
         try {
-            if (
-                (!newConfig?.privateKey || !readFileSync(path.normalize(newConfig?.privateKey!), "utf-8")) &&
-                !newConfig?.password
-            ) {
-                return await promptForPassword(newConfig);
+            const privateKeyPath = newConfig.privateKey;
+
+            if (!newConfig.user) {
+                newConfig.user = await vscode.window.showInputBox({
```

Comment:
Same as the previous comment.

The `newConfig` parameter will contain the `user` when the `validateConfig` function is done.

Do we need it, or can we use the returned object instead? 😋